app/assets/javascripts/expand.js

```diff
@@ -43,30 +43,27 @@ $(document).ready(function() {
         para5.setAttribute("class", "red-completion");
       }
 
-      para5.appendChild(complete_node);
+      para5.innerHTML(complete_node);
```

Comment:
Identifier 'complete_node' is not in camel case.<br>'complete_node' used out of scope.